deploy/olm-catalog/ibm-block-csi-operator-community/1.10.0/manifests/ibm-block-csi-operator.v1.10.0.clusterserviceversion.yaml

```diff
@@ -122,6 +122,41 @@ metadata:
               }
             ]
           }
+        },
+        {
+          "apiVersion": "csi.ibm.com/v1",
+          "kind": "HostDefiner",
+          "metadata": {
+            "name": "host-definer"
+          },
+          "spec": {
+            "hostDefiner": {
+              "repository": "ibmcom/ibm-block-csi-host-definer",
+              "tag": "1.10.0",
+              "imagePullPolicy": "IfNotPresent",
+              "affinity": {
+                "nodeAffinity": {
+                  "requiredDuringSchedulingIgnoredDuringExecution": {
+                    "nodeSelectorTerms": [
+                      {
+                        "matchExpressions": [
+                          {
+                            "key": "kubernetes.io/arch",
+                            "operator": "In",
+                            "values": [
+                              "amd64",
+                              "s390x",
+                              "ppc64le"
+                            ]
+                          }
+                        ]
+                      }
+                    ]
+                  }
+                }
+              }
```

Comment:
since json doesn't support comments, I think we should at least add all fields that have a non-empty default value (including booleans), with their default value, to make it easier to edit

Reply:
done